examples/server/server.cpp

```diff
@@ -1258,8 +1304,8 @@ int main(int argc, char **argv)
         auto lock = llama.lock();
 
         const json body = json::parse(req.body);
-        const std::string content = body.value("content", "");
-        const std::vector<llama_token> tokens = llama_tokenize(llama.ctx, content, false);
+        llama.prompt = body["content"];
```

Comment:
Try to avoid `[]` access because it will create new entries in the dictionary if they are missing.

Reply:
What's the proper error resolution here? Raise an exception? Return an empty string?

Reply:
You can generate from an empty prompt, so it should not be required. We had some trouble with `[]` before, that’s why I bring this up.

Reply:
ok. check the current version.